src/openqaoa-core/optimizers/optimization_methods.py

```diff
@@ -1,29 +1,27 @@
-#   Copyright 2022 Entropica Labs
-#
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
-#
-#       http://www.apache.org/licenses/LICENSE-2.0
-#
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-
 """
```

Comment:
perhaps we could split this file too? I think it can be too big

Reply:
Agreed. I'll create a folder with the same module name and split the functions within the file. That way the code doesnt need to change to accomandate the split.

Reply:
Resolved in commit 2095811f78bc763a210611621e42257fafc31f3a